tools/pkglint/lib/aws-service-official-names.json

```diff
@@ -0,0 +1,88 @@
+{
+  "AWS::AmazonMQ": "Amazon MQ",
+  "AWS::ApiGateway": "Amazon API Gateway",
+  "AWS::AppMesh": "AWS App Mesh",
+  "AWS::AppStream": "Amazon AppStream 2.0",
+  "AWS::AppSync": "AWS AppSync",
+  "AWS::ApplicationAutoScaling": "Application Auto Scaling",
+  "AWS::Athena": "Amazon Athena",
+  "AWS::AutoScaling": "Amazon EC2 Auto Scaling",
+  "AWS::AutoScalingPlans": "AWS Auto Scaling",
+  "AWS::Batch": "AWS Batch",
+  "AWS::Budgets": "AWS Budgets",
+  "AWS::CertificateManager": "Amazon Certificate Manager",
+  "AWS::Cloud9": "AWS Cloud9",
+  "AWS::CloudFormation": "AWS CloudFormation",
+  "AWS::CloudFront": "Amazon CloudFront",
+  "AWS::CloudTrail": "AWS CloudTrail",
+  "AWS::CloudWatch": "Amazon CloudWatch",
+  "AWS::CodeBuild": "AWS CodeBuild",
+  "AWS::CodeCommit": "AWS CodeCommit",
+  "AWS::CodeDeploy": "AWS CodeDeploy",
+  "AWS::CodePipeline": "AWS CodePipeline",
+  "AWS::Cognito": "Amazon Cognito",
+  "AWS::Config": "AWS Config",
+  "AWS::DAX": "Amazon DynamoDB Accelerator",
+  "AWS::DLM": "Amazon Data Lifecycle Manager",
+  "AWS::DMS": "AWS Database Migration Service",
+  "AWS::DataPipeline": "AWS Data Pipeline",
+  "AWS::DirectoryService": "AWS Directory Service",
+  "AWS::DocDB": "Amazon DocumentDB",
+  "AWS::DynamoDB": "Amazon DynamoDB",
+  "AWS::EC2": "Amazon EC2",
+  "AWS::ECR": "Amazon ECR",
+  "AWS::ECS": "Amazon ECS",
+  "AWS::EFS": "Amazon Elastic File System",
+  "AWS::EKS": "Amazon EKS",
+  "AWS::EMR": "Amazon EMR",
+  "AWS::ElastiCache": "Amazon ElastiCache",
+  "AWS::ElasticBeanstalk": "AWS Elastic Beanstalk",
+  "AWS::ElasticLoadBalancing": "Amazon Elastic Load Balancing",
+  "AWS::ElasticLoadBalancingV2": "Amazon Elastic Load Balancing V2",
+  "AWS::Elasticsearch": "Amazon Elasticsearch Service",
+  "AWS::Events": "Amazon CloudWatch Events",
+  "AWS::FSx": "Amazon FSx",
+  "AWS::GameLift": "Amazon GameLift",
+  "AWS::Glue": "AWS Glue",
+  "AWS::Greengrass": "AWS IoT Greengrass",
+  "AWS::GuardDuty": "Amazon GuardDuty",
+  "AWS::IAM": "AWS Identity and Access Management",
+  "AWS::Inspector": "Amazon Inspector",
+  "AWS::IoT1Click": "AWS IoT 1-Click",
+  "AWS::IoT": "AWS IoT",
+  "AWS::IoTAnalytics": "AWS IoT Analytics",
+  "AWS::KMS": "AWS Key Management Service",
+  "AWS::Kinesis": "Amazon Kinesis",
+  "AWS::KinesisAnalytics": "Amazon Kinesis Data Analytics",
+  "AWS::KinesisFirehose": "Amazon Kinesis Data Firehose",
+  "AWS::Lambda": "AWS Lambda",
+  "AWS::Logs": "Amazon CloudWatch Logs",
+  "AWS::MediaStore": "AWS Elemental MediaStore",
+  "AWS::Neptune": "Amazon Neptune",
+  "AWS::OpsWorks": "AWS OpsWorks",
+  "AWS::OpsWorksCM": "AWS OpsWorks CM",
+  "AWS::PinpointEmail": "Amazon Pinpoint Email",
+  "AWS::RAM": "AWS Resource Access Manager",
+  "AWS::RDS": "Amazon Relational Database Service",
+  "AWS::Redshift": "Amazon Redshift",
+  "AWS::RoboMaker": "AWS RoboMaker",
+  "AWS::Route53": "Amazon Route53",
```

Comment:
Can we also enforce that the README's H1 is: `# <official-name> Construct Library`

Reply:
I thought about it. Love the idea - will be looking into that right now.